angular-app/src/app/app.component.ts

```diff
@@ -1,19 +1,34 @@
-import { Component, AfterViewInit, ViewChild } from '@angular/core';
-import { ToggleChatService } from './toggle-chat.service';
+import { Component, OnInit, AfterViewInit, ViewChild } from '@angular/core';
 import { MatSidenav } from '@angular/material/sidenav';
 
+import { ToggleChatService } from './toggle-chat.service';
+import { AuthService } from './auth.service';
+
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent implements AfterViewInit {
+export class AppComponent implements OnInit, AfterViewInit {
+
   @ViewChild('chat') public chat: MatSidenav;
+  public user$;
 
-  constructor(private toggleChatService: ToggleChatService) {
+  constructor(
+    public authService: AuthService,
```

Comment:
Similar question, why does authService need to be public?

Reply:
Oops! This one is actually fine as private; I forgot to change it when I was messing around with other things trying to get it to work. Thanks for the catch!